__tests__/app.test.js

```diff
@@ -45,4 +45,30 @@ describe("/api/articles", () => {
             })
         })
     })
+})
+describe("api/articles/:article_id", () => {
+    test(":) GET 200 - returns article given the article id", () => {
+        return request(app).get("/api/articles/3").expect(200).then(({body}) => {
+            expect(Object.keys(body).length).toBe(7)
+            expect(body).toEqual({
+                "author" : expect.any(String),
+                "title": expect.any(String),
+                "article_id" : 3,
+                "body" : expect.any(String),
+                "topic": expect.any(String),
+                "created_at": expect.any(String),
+                "votes" : expect.any(Number)
+            })
+        })
+    })
+    test(":( GET 400 - returns bad request given invalid ID", () => {
+        return request(app).get("/api/articles/dfs").expect(400).then(({body}) => {
+            console.log(body)
```

Comment:
You're just console.logging the body here, not using it for the test. You should be asserting what the error message is here.